src/components/buttons/AddIdeaButton/AddIdeaButton.js

```diff
@@ -27,7 +28,7 @@ const AddIdeaButton = ({
             setIdeaId(ididea)
             console.log("ideaid", ididea)
             console.log("idea in brackets: ", (createIdeaInDB = {ididea}))
-/*             alert("Idea with id: " + ididea + " is created - HURRRA!") */
+/*          alert("Idea with id: " + ididea + " is created - HURRRA!") */
```

Comment:
Is it possible to style `alert`s? I think that is possible, but we have not considered it in our Figma design, right? 🧐